### DISCUSSION TITLE
Network map

### DISCUSSION THREAD

**Discussion by slrslr** [markqvist/Reticulum#561] [Help & Questions]
I have been unable to find any map of the Reticulum network.
I wanted to see if there is any nearby network that I may join to help building more decentralized network.

**Comment by michael23505** [comment]
i"m new to reticulum so i'll answer as best i can. as far as i know there is not a single complete network to map. there may be several private networks that we have no way of knowing about unless they advertise somehow. there are a few public test networks that are online. amsterdam.connect.reticulum network on port 4965. dublin.connect.reticulum.network on port 4965. reticulum.betweentheborders.com on port 4242. i have quite a few test instances on these that i'd love to chat over and help build the network.

**Comment by LinuxinaBit** [comment]
What @michael23505 said is absolutely correct, though I will add that a map of LoRa nodes connected to the main testnet, made up of user contributions, is a possibility down the line, and something that a lot of people want.

It’s worth noting that the specific servers Michael mentioned are actually interconnected, forming a complete single network (that we call the testnet) along with a ton of other TCP nodes (plus a few LoRa and I2P connected nodes for good measure), though more private networks that are separate do still exist.

**Comment by NeoMatrixJR** [comment]
Just dropping in to say I'd be interested.  I'm seeing fewer nodes online in my area (currently using meshtastic) and I'm wondering if users are exploring alternatives.  Or...get my own started in the area and see if others can be enticed over....

**Comment by n7ekb** [comment]
I think a map would help raise the visibility of Reticulum.  A visible, healthy "public" network should also get the attention of EMCOMM and prepper folks.

**Comment by n7ekb** [comment]
Check out the map that was launched today:  https://rmap.world

**Comment by ghost** [comment]
I'm not a developer (!) so my idea may fail due to things I have no idea about  ¯\\\_(ツ)_/¯

Wouldn't it be nice if the registration process for the map was simplified by sending the telemetry data to a specific collector. In this way, it can be ensured, firstly, that the operator of the node sends this data, and secondly, that it is always up to date, even if the node no longer exists.

**Reply by LinuxinaBit** [reply]
The problem is that telemetry is a very specific thing to require for a map that isn’t even restricted to LXMF…
It’s basically only supported by Sideband.
The best you’re probably going to get is a verification against an LXMF message or some data signed by a specific identity to verify that a node has been added by its actual owner.
That’s not to say using telemetry is a bad idea, and it should certainly be an *option* in the future, but I don’t think it should be mandatory ;)

**Reply by ghost** [reply]
Thank you for the note and the explanation  :+1:

**Reply by fr33n0w** [reply]
on rmap the only "automated" way to add a node at the moment is to add an "mqtt node" , only for Sideband , and is an experimental feature at the moment. you need to enable position telemetry as told by Linuxinabit and Sideband mqtt functionality, connect to the rmap mqtt server and the map will automatically receive your spot (instruction included when adding a mqtt node on the webiste).  these kind of nodes are considered as "live nodes", telemetry is sent by sideband at 60 second interval when connected to mqtt server and when you close sideband, the node disappears from the map after few minutes of inactivity. note: all the enabled telemetry will be sent out to the mqtt server, so enable only the position and maybe some few other telemetry options (light sensor, etc..), or you will see in the node popup all the info sent out from your device.

**Comment by ghost** [comment]
Meanwhile, there is a manual map:  https://rmap.world
and a kind of "Open Collector" node:  `e3b8303e7e2e8c03694f5546264bc3f9`

**Comment by thiswillbeyourgithub** [comment]
Hi, stumbled upon [potatomesh](https://potatomesh.net/) which I think can be a nice UI inspiration for the reticulum world map.
